app/models.py

```diff
@@ -99,7 +99,7 @@ class User(db.Model, ModelMixin, UserMixin):
 
     @classmethod
     def create(cls, email, name, password=None, **kwargs):
-        user: User = super(User, cls).create(email=email, name=name, **kwargs)
```

Comment:
The type hint here is actually for having better auto-completion.

Reply:
I tried and droped mypy quite a long time ago, since then it has improved the SelfType so removal is no longer valid

Reply:
ah ok I see how you got the error then. Yeah I hope `mypy` supports SelfType now. I'm using Pycharm personally and they don't spot any error.